### PR TITLE
replaces the mulebot 16 second hardstun with an 8 second knockdown

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

This PR replaces the 16 second hardstun that non-pAI mulebots with cut avoidance wires apply to living creatures they bump into with a much more tame 8 second knockdown.

## Why It's Good For The Game

unga bunga hardstun bad knockdown good

In all seriousness, there will always be ways to cheese out a "bump" from a hacked non-pAI mulebot, and this PR can act as insurance against/reign in the disgusting power of easy bump methods that might be found in the future and/or might already exist (wink wink nudge nudge).

sorry

## Changelog
:cl: ATHATH
balance: The 16 second hardstun that non-pAI mulebots with cut avoidance wires apply to living creatures they bump into has been replaced with a much more tame 8 second knockdown.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/bot/mulebot.dm [hunk 1]
```diff
@@ -659,7 +659,7 @@
 				if(!paicard)
 					log_combat(src, L, "knocked down")
 					visible_message("<span class='danger'>[src] knocks over [L]!</span>")
-					L.Paralyze(160)
+					L.Knockdown(80)
```
Comment (Qustinnus) [inline code/modules/mob/living/simple_animal/bot/mulebot.dm:662]: 8 SECONDS

**Comment by wesoda25** [comment]
I hate all stuns except this one bc its funny

File: code/modules/mob/living/simple_animal/bot/mulebot.dm (hunk 1 above)
Comment (ATH1909) [reply code/modules/mob/living/simple_animal/bot/mulebot.dm]: thanks

**Comment by yeeyeh** [comment]
genuinely got terrified that you nerfed pai mules for a second there, don't scare me like that dude

**Comment by ATH1909** [comment]
> genuinely got terrified that you nerfed pai mules for a second there, don't scare me like that dude

To be clear, pAI mulebots (with or without this PR) can't hardstun people (or even knock them down) by just walking into them; only non-pAI mulebots can do that (pAI mulebots have to get... "creative").

**Comment by yeeyeh** [comment]
yeah dude I know that I'm one of 3 people who actually know how to do it

**Comment by Tharcoonvagh** [comment]
dude it ain't that complicated

**Comment by ExcessiveUseOfCobblestone** [comment]
does the mulebot really need eight whole seconds for a chance to chain run the person over?

It should be a really small time because all it takes for them to do it again is bump them.

**Comment by PKPenguin321** [comment]
>does the mulebot really need eight whole seconds for a chance to chain run the person over?

Yes.

**Pull request merged**